mediacapture-streams/MediaStream-MediaElement-preload-none-src.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!-- Copyright © 2016 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
+<html>
+    <head>
+        <title>Test that the HTMLMediaElement preload 'none' attribute value is ignored for MediaStream object URLs used as src.</title>>
+        <link rel="author" title="Matthew Wolenetz" href="mailto:wolenetz@chromium.org"/>
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <script src="/common/vendor-prefix.js" data-prefixed-objects='[{"ancestors":["navigator"], "name":"getUserMedia"}, {"ancestors":["window"], "name":"MediaStream"}]'></script>
+    </head>
+    <body>
+        <p class="instructions">When prompted, accept to share your audio stream.</p>
+        <h1 class="instructions">Description</h1>
+        <p class="instructions">This test checks that the HTMLMediaElement preload 'none' attribute value is ignored for MediaStream object URLs used as src.</p>
+
+        <audio id="aud"></audio>
+
+        <script>
+            var aud = document.getElementById("aud");
+            aud.preload="none";
+            var t = async_test("Tests that the HTMLMediaElement preload attribute value 'none' is ignored for MediaStream object URLs used as src.");
+            t.step(function()
+            {
+                navigator.getUserMedia({audio:true}, t.step_func(function(stream)
+                {
+                    // The optional deferred load steps (for preload none) for MediaStream resources should be skipped.
+                    aud.addEventListener("suspend", function() { assert_unreached("'suspend' should not occur here."); }, false);
+                    aud.src = URL.createObjectURL(stream);
+                    assert_equals(aud.networkState, 3); // Resource selection is active.
+
+                    // Let the media element load algorithm proceed, ensure no 'suspend' is fired, and verify that the
+                    // networkState indicates we are loading and not idle. Give it a second.
+                    setTimeout(function() { assert_equals(aud.networkState, 2); t.done(); }, 1000);
+                }), function(error) {});
```

Comment:
Good point. I'll keep the wrapping inside function(error) {...} though since I'm not sure |error| is meaningful as argument to unreached_func. I'll have this fixed later in this PR.